pipelines/broad/rna_seq/BroadInternalRNAWithUMIsPipeline.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+import "RNAWithUMIsPipeline.wdl" as RNAWithUMIsPipeline
+import "../../../tasks/broad/Utilities.wdl" as utils
+
+
+workflow BroadInternalRNAWithUMIsPipeline {
+
+  String pipeline_version = "0.1.0"
+
+  input {
+    # input needs to be either "hg19" or "b38"
+    String reference_build
+
+    # RNAWithUMIsPipeline inputs
+    File? bam
+    File? r1_fastq
+    File? r2_fastq
+    String read1Structure
+    String read2Structure
+    String output_basename
+
+    # only needed if inputs are fastqs instead of ubam
```

Comment:
Are these all required when using fastqs or still optional?

Reply:
These are required for fastq… however for the GP pipeline we can make fastq a required input and remove the BAM input (they will only be pushing fastqs)